release-notes/aws/v9.1.1.md

```diff
@@ -0,0 +1,16 @@
+## :zap: Giant Swarm Release 9.1.1 for AWS is now active for you! :zap:
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
+
+- No change from the previous version.
+
+### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+
+- Allow disabling TCP DNS check.
```

Comment:
```suggestion
- Allowed disabling TCP DNS check.
```

Reply:
Any way the change makes customer's life better? (Figuring out if should write sth for paragraph at top)

Reply:
It made our life better, it's a fix for https://github.com/giantswarm/giantswarm/issues/7898 - our monitoring tool made use of it and was erroring/alarming on one customer environment where this is not permitted.